src/main/java/org/entando/entando/aps/system/init/DatabaseManager.java

```diff
@@ -152,11 +157,57 @@ public SystemInstallationReport installDatabase(SystemInstallationReport report,
         return report;
     }
 
+    private void legacyDatabaseCheck() throws DatabaseMigrationException, SQLException {
+        String[] dataSourceNames = this.extractBeanNames(DataSource.class);
+        for (String dataSourceName : dataSourceNames) {
```

Comment:
A doubt. In this case we are checking all the datasource defined, even the non-default ones. I think it's more correct to check only the default ones (*Port and *Serv), not the ones defined for system extensions

Reply:
Updated